pom.xml

```diff
@@ -1300,13 +1304,12 @@
             <dependency>
                 <groupId>xalan</groupId>
                 <artifactId>xalan</artifactId>
-                <version>2.7.0</version>
+                <version>2.7.2</version>
```

Comment:
FYI, we purposefully downgraded `xalan` to 2.7.0 recently cause of this bug: https://jira.duraspace.org/browse/DS-3733

We probably should add in a comment on this dependency to note that upgrading xalan causes issues with OAI